### DISCUSSION TITLE
Capcha

### DISCUSSION THREAD

**Discussion by mohabaee** [kanboard/kanboard#5215] [Installation & Configuration]
Everywhere there is a Capcha  picture And also icon, logo and etc pictures, it wasn't shown. Also, I uninstalled all plugins and Checked this issue and saw this issue didn't solve. You can see in screenshots that I uploaded on below links:
https://s6.uupload.ir/files/1_oz1u.jpg
https://s6.uupload.ir/files/227_09eq.jpg
https://s6.uupload.ir/files/229_skxp.jpg

### Checklist

- [*] I verified that Kanboard is correctly installed
- [*] I verified that the problem does not come from a plugin
- [* ] I verified that the problem is not already reported
- [ *] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

- Kanboard version: 
- Database type and version:1.2.22
- PHP version: 8.1.6
- OS:windows
- Browser: Chrome

**Comment by fguillot** [comment]
Make sure you have the GD extension installed on your server. Look at the web server log files to determine the root cause of the problem.
